.gitattributes

```diff
@@ -1,3 +1,5 @@
 borg/_version.py export-subst
 
 *.py diff=python
+docs/usage merge=ours
+docs/man merge=ours
```

Comment:
git config --global merge.ours.driver true

is that also needed? (found that on SO)